src/aws_s3_auth.tsx

```diff
@@ -38,10 +38,20 @@ export class GetS3Keys extends React.Component<Props>{
           bucketName: target.bucketName.value
       };
 
-      const s3Auth = new MyS3Auth(awsS3Keys.accessKey, awsS3Keys.secretAccessKey, awsS3Keys.bucketName);
-      // This doesn't work complete, it only prints success or failure to console
-      const isValidUser = s3Auth.checkValidUser();
 
+      // Create a new S3Auth object with the user's keys
+      let s3Auth = new MyS3Auth(awsS3Keys.accessKey, awsS3Keys.secretAccessKey);
```

Comment:
Use const to create the object, you can still modify its properties (change the boolean for valid user, change region, etc)